<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Settings</class>
 <widget class="QMainWindow" name="Settings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>500</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>500</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>500</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>General</string>
       </attribute>
      </widget>
      <widget class="QWidget" name="Calibration_Tab">
       <attribute name="title">
        <string>Calibration</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="0" column="0">
         <widget class="QTableWidget" name="cali_points">
          <property name="inputMethodHints">
           <set>Qt::ImhNone</set>
          </property>
          <property name="alternatingRowColors">
           <bool>true</bool>
          </property>
          <property name="showGrid">
           <bool>true</bool>
          </property>
          <property name="sortingEnabled">
           <bool>true</bool>
          </property>
          <property name="rowCount">
           <number>9</number>
          </property>
          <property name="columnCount">
           <number>3</number>
          </property>
          <attribute name="horizontalHeaderVisible">
           <bool>true</bool>
          </attribute>
          <attribute name="verticalHeaderVisible">
           <bool>false</bool>
          </attribute>
          <attribute name="verticalHeaderHighlightSections">
           <bool>true</bool>
          </attribute>
          <row/>
          <row/>
          <row/>
          <row/>
          <row/>
          <row/>
          <row/>
          <row/>
          <row/>
          <column>
           <property name="text">
            <string>X Pos</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Y Pos</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Z Pos</string>
           </property>
          </column>
          <item row="0" column="0">
           <property name="text">
            <string/>
           </property>
          </item>
          <item row="0" column="1">
           <property name="text">
            <string/>
           </property>
          </item>
          <item row="0" column="2">
           <property name="text">
            <string/>
           </property>
          </item>
          <item row="1" column="0">
           <property name="text">
            <string/>
           </property>
          </item>
          <item row="1" column="1">
           <property name="text">
            <string/>
           </property>
          </item>
          <item row="1" column="2">
           <property name="text">
            <string/>
           </property>
          </item>
          <item row="2" column="0">
           <property name="text">
            <string/>
           </property>
          </item>
          <item row="2" column="1">
           <property name="text">
            <string/>
           </property>
          </item>
          <item row="2" column="2">
           <property name="text">
            <string/>
           </property>
          </item>
          <item row="3" column="0">
           <property name="text">
            <string/>
           </property>
          </item>
          <item row="3" column="1">
           <property name="text">
            <string/>
           </property>
          </item>
          <item row="3" column="2">
           <property name="text">
            <string/>
           </property>
          </item>
          <item row="4" column="0">
           <property name="text">
            <string/>
           </property>
          </item>
          <item row="4" column="1">
           <property name="text">
            <string/>
           </property>
          </item>
          <item row="4" column="2">
           <property name="text">
            <string/>
           </property>
          </item>
          <item row="5" column="0">
           <property name="text">
            <string/>
           </property>
          </item>
          <item row="5" column="1">
           <property name="text">
            <string/>
           </property>
          </item>
          <item row="5" column="2">
           <property name="text">
            <string/>
           </property>
          </item>
          <item row="6" column="0">
           <property name="text">
            <string/>
           </property>
          </item>
          <item row="6" column="1">
           <property name="text">
            <string/>
           </property>
          </item>
          <item row="6" column="2">
           <property name="text">
            <string/>
           </property>
          </item>
          <item row="7" column="0">
           <property name="text">
            <string/>
           </property>
          </item>
          <item row="7" column="1">
           <property name="text">
            <string/>
           </property>
          </item>
          <item row="7" column="2">
           <property name="text">
            <string/>
           </property>
          </item>
          <item row="8" column="0">
           <property name="text">
            <string/>
           </property>
          </item>
          <item row="8" column="1">
           <property name="text">
            <string/>
           </property>
          </item>
          <item row="8" column="2">
           <property name="text">
            <string/>
           </property>
          </item>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QFrame" name="frame_3">
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <layout class="QGridLayout" name="gridLayout_5">
           <item row="1" column="0">
            <widget class="QFrame" name="frame_2">
             <property name="frameShape">
              <enum>QFrame::StyledPanel</enum>
             </property>
             <property name="frameShadow">
              <enum>QFrame::Raised</enum>
             </property>
             <layout class="QGridLayout" name="gridLayout_4">
              <item row="0" column="0">
               <widget class="QLabel" name="label_3">
                <property name="text">
                 <string>Y min</string>
                </property>
               </widget>
              </item>
              <item row="0" column="1">
               <widget class="QLineEdit" name="y_min"/>
              </item>
              <item row="1" column="0">
               <widget class="QLabel" name="label_4">
                <property name="text">
                 <string>Y max</string>
                </property>
               </widget>
              </item>
              <item row="1" column="1">
               <widget class="QLineEdit" name="y_max"/>
              </item>
             </layout>
            </widget>
           </item>
           <item row="0" column="0">
            <widget class="QFrame" name="frame_4">
             <property name="frameShape">
              <enum>QFrame::StyledPanel</enum>
             </property>
             <property name="frameShadow">
              <enum>QFrame::Raised</enum>
             </property>
             <layout class="QGridLayout" name="gridLayout_6">
              <item row="0" column="1">
               <widget class="QLineEdit" name="x_min">
                <property name="inputMethodHints">
                 <set>Qt::ImhNone</set>
                </property>
                <property name="clearButtonEnabled">
                 <bool>false</bool>
                </property>
               </widget>
              </item>
              <item row="0" column="0">
               <widget class="QLabel" name="label">
                <property name="text">
                 <string>X min</string>
                </property>
               </widget>
              </item>
              <item row="1" column="0">
               <widget class="QLabel" name="label_2">
                <property name="text">
                 <string>X max</string>
                </property>
               </widget>
              </item>
              <item row="1" column="1">
               <widget class="QLineEdit" name="x_max"/>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QFrame" name="frame">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <widget class="QPushButton" name="exit_button">
         <property name="text">
          <string>Exit</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QPushButton" name="save_and_exit_button">
         <property name="text">
          <string>Save and Exit</string>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QPushButton" name="save_button">
         <property name="text">
          <string>Save Changes</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
